app/photo.py

```diff
@@ -40,6 +47,20 @@ class SizeTooLargeError(PhotoError):
                 'Please upload a smaller one.')
 
 
+def create_photo_from_url(photo_url, handler):
+    """Creates a photo from a URL.
+
+    Returns:
+      A tuple with an Image and URL with which to serve it, or (None, None) if
+      the image is invalid.
+    """
+    response = requests.get(photo_url)
```

Comment:
Just curious:

* Maybe we should set the user agent header to something appropriate here?
* Maybe we should check robots.txt to see if the photo allows fetch by robots?

Reply:
Are robots.txt files applicable to things other than crawlers? Particularly since we're essentially fetching the file on behalf of a human user, I'm not sure we need to worry about robots.txt files (?).
Any suggestions on the user agent? Can we just make one up (i.e., something with "PersonFinder" in the name)?

Reply:
> Are robots.txt files applicable to things other than crawlers? Particularly since we're essentially fetching the file on behalf of a human user, I'm not sure we need to worry about robots.txt files (?).

I'm not really sure about it, but probably you are right:

http://www.robotstxt.org/faq/what.html
> A robot is a program that automatically traverses the Web's hypertext structure by retrieving a document, and recursively retrieving all documents that are referenced.

Since we are not **traversing** or **recursively** retrieving documents.

> Any suggestions on the user agent? Can we just make one up (i.e., something with "PersonFinder" in the name)?

Yeah I guess we can just make up. Usually they contain the URL of the site e.g.:

Google Person Finder (https://google.org/personfinder)